cypress/integration/todo_spec.js

```diff
@@ -64,4 +65,33 @@ describe('TodoMVC - React', () => {
     todoItems()
       .should('contain', EDITED_ITEM)
   })
+
+  it('should clear completed items', () => {
+    cy.get('.new-todo')
+    .type(NEW_ITEMS[0]).type('{enter}')
```

Comment:
As these items are prerequisite steps (`given` in gherkin), they need to be done without UI interaction. Better to use localStorage for this.